components/Base/Counter.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-  <div class="grid grid-cols-3 place-items-center w-[125px] gap-x-1">
+  <div class="grid w-[125px] grid-cols-3 place-items-center gap-x-1">
```

Comment:
Why with every new PR this formatting changes are coming? Could we have a single PR which will contain formatting of everything at once to have this issue solved? 🤔

Reply:
Branch was created before `tailwind fix` and I have not merged / rebased changes

Reply:
Got it 👌